apps/src/p5lab/P5Lab.js

```diff
@@ -296,6 +297,26 @@ export default class P5Lab {
       // If we reset a puzzle, we should reset the selected poem on that project.
       project.sourceHandler.setSelectedPoem(null);
       this.studioApp_.resetButtonClick();
+      // If a P5 lab project/level that allows custom uploaded images via AnimationPicker was flagged
+      // for abuse due to image moderation (abuse score == 15), starting the project over removes all
+      // user-uploaded images. Therefore, we unflag the project.
+      // Note: once a project is flagged for abuse, the user cannot upload additional custom images.
+      if (
+        ['gamelab', 'poetry', 'spritelab'].includes(
+          project.getStandaloneApp()
+        ) &&
+        project.getAbuseScore() === 15
```

Comment:
should this be >= 15?

Reply:
When the project is flagged for abuse due to image moderation, then the abuse score is set to 15. If a project is flagged for abuse due to a report by a verified teacher or project validator, the score is increased by 20 and if flagged by a non-verified user, the score increases by 10.

When a project meets the abuse threshold (15), uploading images is disabled. I have `===15` for the case if a user's project was flagged by a non-verified teacher user, and then the user uploads a flagged image.
In this case, I didn't want to automatically unblock the project. But maybe we should? I'll ask @dju90 what he thinks. What are your thoughts on this?

Reply:
After thinking about the case of a user who can access a prior version of a project that was flagged due to an abuse report, this brought up the following case which I included in the updated description: uploaded images (via `AnimationPicker`) are NOT deleted from S3, and a user could unblock themself, go back in version history and restore a version that includes the flagged image but the project would NOT be flagged. The reason is that an image is only moderated when it is uploaded.
Thus, I'm removing the ability for a user to unblock themself by starting over.